subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/repositories/DefaultMavenArtifactRepository.java

```diff
@@ -293,7 +293,7 @@ ImmutableMetadataSources createMetadataSources(MavenMetadataLoader mavenMetadata
             sources.add(gradleModuleMetadataSource);
         }
         if (metadataSources.mavenPom) {
-            DefaultMavenPomMetadataSource pomMetadataSource = new DefaultMavenPomMetadataSource(MavenMetadataArtifactProvider.INSTANCE, getPomParser(), fileResourceRepository, getMetadataValidationServices(), mavenMetadataLoader);
+            DefaultMavenPomMetadataSource pomMetadataSource = getPomMetadataSource(mavenMetadataLoader, fileResourceRepository);
```

Comment:
:o: usually I avoid method starting with `get` when they are not getters. `createPomMetadataSource`

Reply:
And this rings a bell ... not the first time I get the remark ... which I agree with ... fixing 😉